db/migrate/20201128072303_add_timestamps_to_framework_frameworks.rb

```diff
@@ -0,0 +1,5 @@
+class AddTimestampsToFrameworkFrameworks < ActiveRecord::Migration[5.0]
+  def change
+    add_timestamps :frameworks_frameworks
```

Comment:
@nrankin should be `:framework_frameworks`?

Reply:
yeah, I'll leave the old migrations alone tho :)